src/coreclr/tools/superpmi/superpmi/CMakeLists.txt

```
@@ -12,6 +12,15 @@ if(CLR_CMAKE_HOST_WIN32)
   set(CMAKE_MSVC_RUNTIME_LIBRARY MultiThreaded)
 endif(CLR_CMAKE_HOST_WIN32)
 
+if(CLR_CMAKE_HOST_UNIX)
+    # Make sure a public symbol is created for Instrumentor_GetInsCount
+    if(CLR_CMAKE_HOST_APPLE)
+        add_linker_flag(-Wl,-export_dynamic)
+    else()
+        add_linker_flag(-Wl,--export-dynamic)
+    endif(CLR_CMAKE_HOST_APPLE)
```

Looks like maybe you could also just export `Instrumentor_GetInsCount` instead of everything:

```suggestion
    if(CLR_CMAKE_HOST_APPLE)
        add_linker_flag(-Wl,-exported_symbol Instrumentor_GetInsCount)
    else()
        add_linker_flag(-Wl,--export-dynamic-symbol=Instrumentor_GetInsCount)
    endif(CLR_CMAKE_HOST_APPLE)
```

---

`--export-dynamic-symbol` is relatively new, my `ld` does not have it. Maybe it's ok for the Apple case?

---

Might as well go with the simple "export all" solution then (I'm assuming it doesn't cause any problems to export "too much")